packages/synapse-react-client/esbuild.config.mjs

```diff
@@ -61,22 +48,8 @@ const esBuildOptions = {
     'react-bootstrap',
     'react-plotly.js/factory',
     'plotly.js-basic-dist',
-    'katex',
     'rss-parser',
     'react-mailchimp-subscribe',
-    'markdownit',
-    'markdownitSynapse',
-    'markdownitSub',
-    'markdownitSup',
-    'markdownitCentertext',
-    'markdownitSynapseHeading',
-    'markdownitSynapseTable',
-    'markdownitStrikethroughAlt',
-    'markdownitContainer',
-    'markdownitEmphasisAlt',
-    'markdownitInlineComments',
-    'markdownitBr',
-    'markdownitMath',
```

Comment:
They previously loaded one copy of each, stored in a global variable, which was used by SRC and GWT.

With this change, they will be included in the UMD bundle. Though now that I think about it, I think we can keep this instead of changing it.